applications/substrate-identity-directory.md

```diff
@@ -103,9 +97,23 @@ We have experience in several private and open source projects aswell as project
 | 2. | Make the web service work offline | Web service can be used in offline mode; the user can specify a local node to which will the service connect. |
  
 
+### Milestone 3 Index data and query
+* **Estimated Duration:** 3 weeks
+* **FTE:**  2
+* **Costs:** 8166 DAI
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 1. | Index data from nodes | Setup [substrate-archive](https://github.com/paritytech/substrate-archive) with postgres for Polkadot and Kusama chains to store required data |
```

Comment:
We also only fund deliverables that may be useful to others in the ecosystem, i.e. we can't support you in integrating existing software.

Reply:
I am not sure if I understand. This backend infrastructure is a requirement for governance & treasury data to be retrieved.